docs/general/guides/communications/communications.md

```diff
@@ -0,0 +1,114 @@
+# Communications
+
+You can send custom messages to voters at any time during the electoral process, from
+the election page in the admin console. It can be used to send authentication codes for
```

Comment:
The "election page" mention is not clear. These messages can be sent from the Election Dashboard  and to specific voters from the Election's Census Data section.